Dell/recovery_common.py

```diff
@@ -640,6 +640,31 @@ def _h_error(exception=None):
         raise _h_exception_exc
     return _h_reply_result
 
+def regenerate_md5sum(fd,fs=None):
+'''regenerate the md5sum.txt when building the ISO image
+
+    This function is based on the md5sum.txt exsit and some content has changed.
+    If the file does not exist before,it will turn to fail.
+    '''
+    pattern = re.compile("[0-9a-z]{32}  \.")
+    with open(fd,'r') as rfd:
+        source_lines = rfd.readlines()
+    with open(fd,'w') as wfd:
+        for line in source_lines:
```

Comment:
So this will only update md5sum for files that have been changed that already existed.  If possible, any _new_ files that are found on the HDD should also be included here too.  For example new files added by add-on packages